src/useq/_mda_sequence.py

```diff
@@ -323,21 +357,41 @@ def iter_sequence(sequence: MDASequence) -> Iterator[MDAEvent]:
     order = sequence.used_axes
 
     event_iterator = (enumerate(sequence.iter_axis(ax)) for ax in order)
+
     for global_index, item in enumerate(product(*event_iterator)):
         if not item:  # the case with no events
             continue
 
         _ev = dict(zip(order, item))
+
         index = {k: _ev[k][0] for k in INDICES if k in _ev}
 
         position: Optional[Position] = _ev[POSITION][1] if POSITION in _ev else None
         channel: Optional[Channel] = _ev[CHANNEL][1] if CHANNEL in _ev else None
         time: Optional[int] = _ev[TIME][1] if TIME in _ev else None
+        tile: Optional[dict[str, bool | float]] = _ev[TILE][1] if TILE in _ev else None
+        # tile e.g. {'is_relative': True, 'dx': 0, 'dy': 0}
+        # or {'is_relative': False, 'x': 0, 'y': 0}
 
         # skip channels
         if channel and TIME in index and index[TIME] % channel.acquire_every:
             continue
 
+        x_pos = getattr(position, "x", None)
+        y_pos = getattr(position, "y", None)
+
+        if tile:
+            if tile.get("is_relative"):
+                # e.g. TileRelative
+                dx = tile.get("dx")
```

Comment:
don't use `.get` in a dict unless `None` is a valid value in later lines.  Here, it's not... since you can't add `x_pos` to `None`.  If the value _must_ be there use `tile['dx']`

Reply:
also... I would just use `x` for both relative and not relative.  We already have the "is_relative" key that tells us the meaning of x ... we don't also need to rename it